scripts/migrate_script.py

```diff
@@ -113,7 +113,7 @@ def collect_stdlib_packages() -> Tuple[List[StdLibPackage], List[StdLibPackage]]
     add_stdlib_packages_from("stdlib/2and3", stdlib, "2.7")
     # Use oldest currently supported version for Python 3 packages/modules.
     add_stdlib_packages_from("stdlib/3", stdlib, DEFAULT_PY3_VERSION)
-    for version in ("3.6", "3.7", "3.8", "3.9"):
+    for version in ("3.7", "3.8", "3.9"):
```

Comment:
This looks to be removing 3.6, not 3.5. Is there a reason for not support stdlib3.6 when removing 3.5?

Reply:
This iterates over the directories in strong/third-party. But the 3.6 directory has now been merged into the 3 directory.

Reply:
Ahh, right